app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -53,21 +63,110 @@ class SuggestedEditsTasksFragment : Fragment() {
+        val topDecorationDp = PADDING_16
         tasksRecyclerView.addItemDecoration(FooterMarginItemDecoration(0, topDecorationDp))
+        setUpTasks()
         tasksRecyclerView.adapter = RecyclerAdapter(displayedTasks)
+    }
 
-        usernameText.text = AccountUtil.getUserName()
-        userContributionsButton.setOnClickListener {
-            startActivity(SuggestedEditsContributionsActivity.newIntent(requireContext()))
+    private fun onUserStatClicked(view: View) {
+        when (view) {
+            contributionsStatsView -> showContributionsStatsViewTooltip()
+            editStreakStatsView -> showEditStreakStatsViewTooltip()
+            pageViewStatsView -> showPageViewStatsViewTooltip()
+            else -> showEditQualityStatsViewTooltip()
+        }
+    }
+
+    private fun showContributionsStatsViewTooltip() {
+        tooltipLayout.visibility = VISIBLE
+        val param = topTooltipArrow.layoutParams as LinearLayout.LayoutParams
+        param.gravity = Gravity.START
+        topTooltipArrow.layoutParams = param
+        tooltipTextView.text = getString(R.string.suggested_edits_contributions_stat_tooltip)
+        executeAfterTimer(true)
+    }
+
+    private fun showEditStreakStatsViewTooltip() {
+        tooltipLayout.visibility = VISIBLE
+        val param = topTooltipArrow.layoutParams as LinearLayout.LayoutParams
+        param.gravity = Gravity.END
+        topTooltipArrow.layoutParams = param
+        tooltipTextView.text = getString(R.string.suggested_edits_edit_streak_stat_tooltip)
+        executeAfterTimer(true)
+    }
+
+    private fun showPageViewStatsViewTooltip() {
+        bottomTooltipArrow.visibility = VISIBLE
+        val param = bottomTooltipArrow.layoutParams as LinearLayout.LayoutParams
+        param.gravity = Gravity.START
+        bottomTooltipArrow.layoutParams = param
+        textViewForMessage.setBackgroundColor(ResourceUtil.getThemedColor(context!!, R.attr.paper_color))
+        textViewForMessage.elevation = ELEVATION_4
+        textViewForMessage.setPadding(PADDING_16, PADDING_16, PADDING_16, PADDING_16)
+        textViewForMessage.text = getString(R.string.suggested_edits_page_views_stat_tooltip)
+        executeAfterTimer(false)
+    }
+
+    private fun showEditQualityStatsViewTooltip() {
+        bottomTooltipArrow.visibility = VISIBLE
+        val param = bottomTooltipArrow.layoutParams as LinearLayout.LayoutParams
+        param.gravity = Gravity.END
+        bottomTooltipArrow.layoutParams = param
+        textViewForMessage.setBackgroundColor(ResourceUtil.getThemedColor(context!!, R.attr.paper_color))
+        textViewForMessage.elevation = ELEVATION_4
+        textViewForMessage.setPadding(PADDING_16, PADDING_16, PADDING_16, PADDING_16)
+        textViewForMessage.text = getString(R.string.suggested_edits_edit_quality_stat_tooltip, 3)
+        executeAfterTimer(false)
+    }
+
+    private fun executeAfterTimer(isTopTooltip: Boolean) {
```

Comment:
Can we hide other tooltips before showing a new tooltip? If we click on either "Pageview" or "Edit quality" and quickly click on the top two icons, there will have two tooltips showing in the same time.